data/assets/default.scene

```diff
@@ -10,6 +10,13 @@ asset.onInitialize(function ()
     openspace.globebrowsing.goToGeo("Earth", 58.5877, 16.1924, 20000000)
 
     openspace.markInterestingNodes({ "Earth", "Mars", "Moon", "Sun" })
+
+     -- HDR / Image options:
+    openspace.setPropertyValueSingle('RenderEngine.Gamma', 0.95);
+    openspace.setPropertyValueSingle('RenderEngine.HDRExposure', 3.7);
+    openspace.setPropertyValueSingle('RenderEngine.ToneMapOperator', 8);
+    openspace.setPropertyValueSingle('RenderEngine.Lightness', 1.1);
+    openspace.setPropertyValueSingle('RenderEngine.Saturation', 1.2);
```

Comment:
Isn't it strange that the default behavior is to boost up the colors?
I think it would make sense if pretty much all the settings were set to 1.0, and then adjust individual renderables to look good with this default setting.